R/format-table.R

```diff
@@ -128,4 +107,38 @@ check_values_for_table <-
         }
       }
     }
-  }
\ No newline at end of file
+  }
+
+#' Format part table
+#' 
+#' A wrapper function that combines format_table and check_values_for_table. 
+#' Both formatting the table and displaying table data specific warnings.
+#' 
+#' @param input_table data.frame containing the parts table
+#' @param column_names string vector containing columns to check
+#' 
+#' @return data.frame containing the formated parts table
+format_parts_table <- function(input_table, column_names) {
```

Comment:
This should be moved into its own file. Ideally, it should be in the same module as the `parts-module.R` file in the other [PR](https://github.com/Big-Life-Lab/PHES-ODM-Doc/pull/37).

Reply:
`input_table` can be renamed to `parts_table`

Reply:
Does the user of this function need to pass in `column_names`? Can we not calculate it within this function?